os/arch/arm/src/armv7-a/arm_createstack.c

```diff
@@ -194,6 +194,7 @@ int up_create_stack(struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
       size_of_stack = top_of_stack - (uintptr_t)tcb->stack_alloc_ptr;
 
       /* Save the adjusted stack values in the struct tcb_s */
+		tcb->adj_stack_ptr = (uint32_t *)top_of_stack;
```

Comment:
You use spaces at other lines. Let's keep the same way in this PR. And let's modify all of coding rules violations with new PR.

Reply:
We will take up coding rule violations in a separate PR.